Quaver.Shared/Screens/Settings/SettingsDialog.cs

```diff
@@ -351,6 +351,7 @@ private void CreateSections()
                     new SettingsScrollDirection(this, "Scroll Direction 4K", ConfigManager.ScrollDirection4K),
                     new SettingsScrollDirection(this, "Scroll Direction 7K", ConfigManager.ScrollDirection7K),
                     new SettingsBool(this, "Display Timing Lines", ConfigManager.DisplayTimingLines),
+                    new SettingsBool(this, "Display Snap Color Overrides", ConfigManager.DisplaySnapColorOverrides),
```

Comment:
I think the code here was part of the settings menu for alpha, and isn't actually used for the current builds

Reply:
You are correct, it got me as well and had me stuck for 30min when I worked on tooltips